apps/meteor/app/lib/server/functions/deleteUser.ts

```diff
@@ -25,7 +26,9 @@ export async function deleteUser(userId: string, confirmRelinquish = false): Pro
 		projection: { username: 1, avatarOrigin: 1, roles: 1, federated: 1 },
 	});
 
-	if (!user) {
+	const actionUserId = Meteor.userId();
+
+	if (!user || !actionUserId) {
```

Comment:
pass the userid as an argument, avoid using meteor.

Reply:
That would require modifying the existing `deleteUser` function, and so potentially I will have to modify the files which call it, would that be ok?

Reply:
Here we need the uid of the user who is calling this function. Because while hiding the reported messages, the moderator's id is stored along with the action.

If a user deletes their own account, we can use the passed `user._id`, but if someone else calls the `deleteUser`, the userId of the user whose account is going to be deleted is passed on, and we don't need that. 

I hope that makes things clear. Thank you!